client/python/test/test_client.py

```diff
@@ -791,6 +782,12 @@ def test_unnamed_endpoints_use_fn_index(self, count_generator_demo):
             assert "fn_index=0" in info
             assert "api_name" not in info
 
+    def test_api_false_endpoints_do_not_appear(self, count_generator_demo):
+        # This demo has no api_name
+        with connect(count_generator_demo) as client:
+            info = client.view_api(return_format="dict")
+            assert len(info["named_endpoints"]) + len(info["unnamed_endpoints"]) == 2
```

Comment:
Wouldn't `assert len(info["unnamed_endpoints"]) == 2 and len(info["named_endpoints"]) == 0 ` be better?

Reply:
Yea

Reply:
Don't know why I wrote it this way